### PR TITLE
Links from workflows to schemas

### PR DESCRIPTION
**Pull request by nissimsan**
This adds links from steel and ecomm workflows to relevant schemas. 

Addresses issues:
- https://github.com/w3c-ccg/traceability-vocab/issues/350
- https://github.com/w3c-ccg/traceability-vocab/issues/351

(Links will be outdated by https://github.com/w3c-ccg/traceability-vocab/pull/599, but that's a simple subsequent fix)

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -756,24 +756,24 @@ <h5>Mill Test Certification</h5>
...
         </p>
         <p>
-          Steel Manufacturer issues a Commercial Invoice as part of a sales transaction and an Intent To Import to initiate the import (4). The documents are presented to the customer, Steel Importer (5), which can verify the originality of both documents, respectively tracing the Commercial Invoice back to the Manufacturer (6) and the Mill Test Report back to a trusted steel certifying agent (7). 
+          Steel Manufacturer issues a <a href="https://w3id.org/traceability/#CommercialInvoiceCertificate">Commercial Invoice</a> as part of a sales transaction and an <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> to initiate the import (4). The documents are presented to the customer, Steel Importer (5), which can verify the originality of both documents, respectively tracing the <a href="https://w3id.org/traceability/#IntentToImportCertificate">Commercial Invoice</a> back to the Manufacturer (6) and the <a href="https://w3id.org/traceability/#MillTestReportCertificate">Mill Test Report</a> back to a trusted steel certifying agent (7).
```
Comment (TallTed) [inline docs/index.html:762]:
```suggestion
          Steel Manufacturer issues a <a href="https://w3id.org/traceability/#CommercialInvoiceCertificate">Commercial Invoice</a> as part of a sales transaction and an <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> to initiate the import (4). The documents are presented to the customer, Steel Importer (5), which can verify the originality of both documents, respectively tracing the <a href="https://w3id.org/traceability/#CommercialInvoiceCertificate">Commercial Invoice</a> back to the Manufacturer (6) and the <a href="https://w3id.org/traceability/#MillTestReportCertificate">Mill Test Report</a> back to a trusted steel certifying agent (7). 
```

File: docs/index.html [hunk 2]
```diff
@@ -756,24 +756,24 @@ <h5>Mill Test Certification</h5>
...
           <img src="resources/sima-workflow.png" alt="SIMA License Workflow" style="width:100%">
           <figcaption>SIMA Steel License Workflow</figcaption>
         </figure>
         <p>
-          Steel Inc applies for a SIMA Licence, issuing (1) and presenting an application to SIMA (2).
+          Steel Inc applies for a <a href="https://w3id.org/traceability/#SIMASteelImportLicenseCertificate">SIMA Licence</a>, issuing (1) and presenting a <a href="https://w3id.org/traceability/#SIMASteelImportLicenseApplicationCertificate">SIMA Steel License Application</a> to SIMA (2).
         </p>
         <p>
-          Steel Import Monitoring and Analysis verifies (3) and processes Steel Inc’s application, issues a SIMA Certificate (4) which is presented back to Steel Inc. (5).
+          Steel Import Monitoring and Analysis verifies (3) and processes Steel Inc’s application, issues a <a href="https://w3id.org/traceability/#SIMASteelImportLicenseCertificate">SIMA Steel License Certificate</a> (4) which is presented back to Steel Inc. (5).
```
Comment (TallTed) [inline docs/index.html:773]:
```suggestion
          Steel Import Monitoring and Analysis verifies (3) and processes Steel Inc’s application, issues a <a href="https://w3id.org/traceability/#SIMASteelImportLicenseCertificate">SIMA Steel License</a> (4) which is presented back to Steel Inc. (5).
```

File: docs/index.html [hunk 3]
```diff
@@ -756,24 +756,24 @@ <h5>Mill Test Certification</h5>
...
         </p>
         <p>
-          At time of import, Steel Inc. issues an Intent To Import and Entry Summary credentials (6) which are presented to CBP together with the SIMA License (7).
+          At time of import, Steel Inc. issues an <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> and <a href="https://w3id.org/traceability/#CBP7501EntrySummaryCertificate">7501 Entry Summary</a> credentials (6) which are presented to CBP together with the <a href="https://w3id.org/traceability/#SIMASteelImportLicenseCertificate">SIMA License</a> (7).
```
Comment (TallTed) [inline docs/index.html:776]:
```suggestion
          At time of import, Steel Inc. issues an <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> and a <a href="https://w3id.org/traceability/#CBP7501EntrySummaryCertificate">7501 Entry Summary</a> (6) which are presented to CBP together with the <a href="https://w3id.org/traceability/#SIMASteelImportLicenseCertificate">SIMA License</a> (7).
```

File: docs/index.html [hunk 4]
```diff
@@ -784,24 +784,24 @@ <h5>Steel Import Under USMCA Free Trade Agreement</h5>
         <figcaption>USMCA Steel Import Workflow</figcaption>
       </figure>
       <p>
-        Steel Importer issues USCMA Certificate (1). Initiating the Importing workflow, Steel Importer also issues an Intent To Import (2). 
+        Steel Importer issues a <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USCMA Certificate</a> (1). Initiating the Importing workflow, Steel Importer also issues an <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> credential (2).
```
Comment (TallTed) [inline docs/index.html:787]:
```suggestion
        Steel Importer issues a <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USCMA Certificate of Origin</a> (1). Initiating the importing workflow, Steel Importer also issues an <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> (2). 
```

File: docs/index.html [hunk 5]
```diff
@@ -784,24 +784,24 @@ <h5>Steel Import Under USMCA Free Trade Agreement</h5>
         <figcaption>USMCA Steel Import Workflow</figcaption>
       </figure>
       <p>
-        Steel Importer issues USCMA Certificate (1). Initiating the Importing workflow, Steel Importer also issues an Intent To Import (2). 
+        Steel Importer issues a <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USCMA Certificate</a> (1). Initiating the Importing workflow, Steel Importer also issues an <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> credential (2). 
       </p>
       <p>
-        Steel Importer presents the USCMA and Intent To Import certificates along with a MTR Certificate to Customs (3).
+        Steel Importer presents the <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USMCA</a> and <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> credentials along with a <a href="https://w3id.org/traceability/#MillTestReportCertificate">Mill Test Report</a> Certificate to Customs (3).
```
Comment (TallTed) [inline docs/index.html:790]:
```suggestion
        Steel Importer presents the <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USMCA Certificate of Origin</a> and <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> along with a <a href="https://w3id.org/traceability/#MillTestReportCertificate">Mill Test Report</a> to Customs (3).
```

File: docs/index.html [hunk 6]
```diff
@@ -784,24 +784,24 @@ <h5>Steel Import Under USMCA Free Trade Agreement</h5>
...
       </p>
       <p>
-        Customs verifies the USMCA (4), cross checks HTS codes with the Mill Test Report (5) and initiates the import procedures (6). 
+        Customs verifies the <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USMCA</a> (4), cross checks HTS codes with the <a href="https://w3id.org/traceability/#MillTestReportCertificate">Mill Test Report</a> (5) and initiates the import procedures (6).
```
Comment (TallTed) [inline docs/index.html:793]:
```suggestion
        Customs verifies the <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USMCA Certification of Origin</a> (4), cross checks HTS codes with the <a href="https://w3id.org/traceability/#MillTestReportCertificate">Mill Test Report</a> (5), and initiates the import procedures (6). 
```

File: docs/index.html [hunk 7]
```diff
@@ -784,24 +784,24 @@ <h5>Steel Import Under USMCA Free Trade Agreement</h5>
...
       <figure>
         <img src="resources/document-pouch-workflow.png" alt="Steel Import Customs Brokerage Workflow" style="width:100%">
         <figcaption>Steel Import Customs Brokerage Workflow</figcaption>
       </figure>
       <p>
-        Steel Importer issues Commercial Invoice (1) and Packing List (2) in relation to a shipment of steel. These are presented to the Customs Broker along a SIMA License, a USMCA Declaration, a Mill Test Report and a CTPAT Importer Certificate. 
+        Steel Importer issues <a href="https://w3id.org/traceability/#CommercialInvoiceCertificate">Commercial Invoice</a> (1) and <a href="https://w3id.org/traceability/#PackingListCertificate">Packing List</a> (2) in relation to a shipment of steel. These are presented to the Customs Broker along a <a href="https://w3id.org/traceability/#SIMASteelImportLicenseCertificate">SIMA License</a>, a <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USMCA Declaration</a>, a <a href="https://w3id.org/traceability/#MillTestReportCertificate">Mill Test Report</a> and a <a href="https://w3id.org/traceability/#CTPATCertificate">CTPAT Importer</a> Certificate.
```
Comment (TallTed) [inline docs/index.html:801]:
```suggestion
        Steel Importer issues <a href="https://w3id.org/traceability/#CommercialInvoiceCertificate">Commercial Invoice</a> (1) and <a href="https://w3id.org/traceability/#PackingListCertificate">Packing List</a> (2) in relation to a shipment of steel. These are presented to the Customs Broker along with a <a href="https://w3id.org/traceability/#SIMASteelImportLicenseCertificate">SIMA License</a>, a <a href="https://w3id.org/traceability/#USMCACertificationOfOrigin">USMCA Declaration</a>, a <a href="https://w3id.org/traceability/#MillTestReportCertificate">Mill Test Report</a>, and a <a href="https://w3id.org/traceability/#CTPATCertificate">CTPAT Importer Certificate</a>. 
```

File: docs/index.html [hunk 8]
```diff
@@ -784,24 +784,24 @@ <h5>Steel Import Under USMCA Free Trade Agreement</h5>
...
       </p>
       <p>
-        The Customs Broker issues 3461 Immediate Delivery (3) and Intent To Import certificates (4), and has received the Bill of Lading from the carrier. 
+        The Customs Broker issues <a href="https://w3id.org/traceability/#CBP3461ImmediateDeliveryCertificate">3461 Immediate Delivery</a> (3) and <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> credentials (4), and has received the <a href="https://w3id.org/traceability/#BillOfLadingCertificate">Bill of Lading</a> from the carrier.
```
Comment (TallTed) [inline docs/index.html:804]:
```suggestion
        The Customs Broker issues a <a href="https://w3id.org/traceability/#CBP3461ImmediateDeliveryCertificate">3461 Immediate Delivery Certificate</a> (3) and an <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> (4), and has received the <a href="https://w3id.org/traceability/#BillOfLadingCertificate">Bill of Lading</a> from the carrier. 
```

File: docs/index.html [hunk 9]
```diff
@@ -913,7 +913,7 @@ <h5>De Minimis Shipment</h5>
           <figcaption>De Minimis Shipment Workflow</figcaption>
         </figure>
         <p>
-          As part of an international ecommerce sale, the Ecommerce platform issues a De Minimis Shipment credential (1) indicating importing duties exemption. An Intent To Import credential is also issued (2) for initiating the import workflow (3). 
+          As part of an international ecommerce sale, the Ecommerce platform issues a <a href="https://w3id.org/traceability/#DeMinimisShipmentCertificate">De Minimis Shipment</a> credential (1) indicating importing duties exemption. An <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> credential is also issued (2) to initiate the import workflow (3).
```
Comment (TallTed) [inline docs/index.html:916]:
```suggestion
          As part of an international ecommerce sale, the Ecommerce platform issues a <a href="https://w3id.org/traceability/#DeMinimisShipmentCertificate">De Minimis Shipment Certificate</a> (1) indicating exemption from importing duties. An <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> is also issued (2) to initiate the import workflow (3). 
```

File: docs/index.html [hunk 10]
```diff
@@ -913,7 +913,7 @@ <h5>De Minimis Shipment</h5>
           <figcaption>De Minimis Shipment Workflow</figcaption>
         </figure>
         <p>
-          As part of an international ecommerce sale, the Ecommerce platform issues a De Minimis Shipment credential (1) indicating importing duties exemption. An Intent To Import credential is also issued (2) for initiating the import workflow (3). 
+          As part of an international ecommerce sale, the Ecommerce platform issues a <a href="https://w3id.org/traceability/#DeMinimisShipmentCertificate">De Minimis Shipment</a> credential (1) indicating importing duties exemption. An <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> credential is also issued (2) to initiate the import workflow (3). 
         </p>
         <p>
           Customs verifies both credentials (4) and (5) and allows the shipment direct pass through customs.
```
Comment (TallTed) [inline docs/index.html:919]:
```suggestion
          Customs verifies both (4) and (5) and allows the shipment "direct pass" through customs. 
```

File: docs/index.html [hunk 11]
```diff
@@ -924,27 +924,27 @@ <h5>Packing and Shipping</h5>
           <figcaption>Packing and Shipping Workflow</figcaption>
         </figure>
         <p>
-          A purchase is made on a US eCommerce platform triggering issuance of a Purchase Order (1) which is passed to the foreign fulfillment center (2).
+          A purchase is made on a US eCommerce platform triggering issuance of a <a href="https://w3id.org/traceability/#PurchaseOrderCertificate">Purchase Order</a> (1) which is passed to the foreign fulfillment center (2).
```
Comment (TallTed) [inline docs/index.html:927]:
```suggestion
          A purchase is made on a US eCommerce platform, triggering issuance of a <a href="https://w3id.org/traceability/#PurchaseOrderCertificate">Purchase Order</a> (1), which is passed to the foreign fulfillment center (2).
```

File: docs/index.html [hunk 12]
```diff
@@ -924,27 +924,27 @@ <h5>Packing and Shipping</h5>
...
         </p>
         <p>
-          Receiving and verifying the Purchase Order, the fulfillment center organizes shipping with an international carrier, issuing (4) and presenting (5) Shipping Instructions.
+          Receiving and verifying the <a href="https://w3id.org/traceability/#PurchaseOrderCertificate">Purchase Order</a>, the fulfillment center organizes shipping with an international carrier, issuing (4) and presenting (5) <a href="https://w3id.org/traceability/#ShippingInstructionsCertificate">Shipping Instructions</a>.
```
Comment (TallTed) [inline docs/index.html:930]:
```suggestion
          Receiving and verifying the <a href="https://w3id.org/traceability/#PurchaseOrderCertificate">Purchase Order</a>, the fulfillment center organizes shipping with an international carrier, issuing (4) and presenting <a href="https://w3id.org/traceability/#ShippingInstructionsCertificate">Shipping Instructions</a> (5).
```

File: docs/index.html [hunk 13]
```diff
@@ -924,27 +924,27 @@ <h5>Packing and Shipping</h5>
...
         </p>
         <p>
-          Upon shipping, the Fulfillment Center issues a Packing List (7), presented to the courier (8). The express courier verifies the Shipping Instructions (6) and prepares for shipping. 
+          Upon shipping, the Fulfillment Center issues a <a href="https://w3id.org/traceability/#PackingListCertificate">Packing List</a> (7), presented to the courier (8). The express courier verifies the <a href="https://w3id.org/traceability/#ShippingInstructionsCertificate">Shipping Instructions</a> (6) and prepares for shipping.
```
Comment (TallTed) [inline docs/index.html:933]:
```suggestion
          Upon shipping, the Fulfillment Center issues a <a href="https://w3id.org/traceability/#PackingListCertificate">Packing List</a> (7), presented to the courier (8). The courier verifies the <a href="https://w3id.org/traceability/#ShippingInstructionsCertificate">Shipping Instructions</a> (6) and prepares for shipping. 
```

File: docs/index.html [hunk 14]
```diff
@@ -924,27 +924,27 @@ <h5>Packing and Shipping</h5>
...
+          The carrier verifies the <a href="https://w3id.org/traceability/#PackingListCertificate">Packing List</a> (9) and automatically correlates it to the shipment through the common workflow instance id.
         </p>
         <h5>Importing Procedures</h5>
         <figure>
           <img src="resources/import-procedures-workflow.png" alt="Import Procedures Workflow" style="width:100%">
           <figcaption>Import Procedures Workflow</figcaption>
         </figure>
         <p>
-          Express Carrier issues House Bill of Lading (1) which is presented along with other documents associated to the Packing List (2) to the Customs Broker. 
+          Express Carrier issues a <a href="https://w3id.org/traceability/#HouseBillOfLadingCertificate">House Bill of Lading</a> (1) which is presented along with other documents associated to the <a href="https://w3id.org/traceability/#PackingListCertificate">Packing List</a> (2) to the Customs Broker.
```
Comment (TallTed) [inline docs/index.html:944]:
```suggestion
          Express Carrier issues a <a href="https://w3id.org/traceability/#HouseBillOfLadingCertificate">House Bill of Lading</a> (1), which is presented to the Customs Broker along with other documents associated with the <a href="https://w3id.org/traceability/#PackingListCertificate">Packing List</a> (2). 
```

File: docs/index.html [hunk 15]
```diff
@@ -924,27 +924,27 @@ <h5>Packing and Shipping</h5>
...
         </figure>
         <p>
-          Express Carrier issues House Bill of Lading (1) which is presented along with other documents associated to the Packing List (2) to the Customs Broker. 
+          Express Carrier issues a <a href="https://w3id.org/traceability/#HouseBillOfLadingCertificate">House Bill of Lading</a> (1) which is presented along with other documents associated to the <a href="https://w3id.org/traceability/#PackingListCertificate">Packing List</a> (2) to the Customs Broker. 
         </p>
         <p>
-          The Customs Broker Issues 3461 Immediate Delivery (3) and Intent To Import (4) certificates. The documents are submitted to Customs through a Traceable Presentation with a defined workflow instance id “a123-b456” (5).
+          The Customs Broker issues a <a href="https://w3id.org/traceability/#CBP3461ImmediateDeliveryCertificate">3461 Immediate Delivery</a> (3) and <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> (4) certificates. The documents are submitted to Customs through a Traceable Presentation with a defined workflow instance id “a123-b456” (5).
```
Comment (TallTed) [inline docs/index.html:947]:
```suggestion
          The Customs Broker issues <a href="https://w3id.org/traceability/#CBP3461ImmediateDeliveryCertificate">3461 Immediate Delivery</a> (3) and <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> (4) certificates. The documents are submitted to Customs through a Traceable Presentation with a defined workflow instance id such as “a123-b456” (5).
```

File: docs/index.html [hunk 16]
```diff
@@ -955,21 +955,21 @@ <h5>Import Completion</h5>
           <figcaption>Import Procedures Workflow</figcaption>
         </figure>
         <p>
-          The Customs Broker completes the importing procedures by issuing the 7501 Entry Summary (1), presenting it to Customs (2), associating it to the earlier submission by correlating the workflow instance id ("a123-b456").
+          The Customs Broker completes the importing procedures by issuing the <a href="https://w3id.org/traceability/#CBP7501EntrySummaryCertificate">7501 Entry Summary</a> (1), presenting it to Customs (2), associating it to the earlier submission by correlating the workflow instance id ("a123-b456").
```
Comment (TallTed) [inline docs/index.html:958]:
```suggestion
          The Customs Broker completes the importing procedures by issuing the <a href="https://w3id.org/traceability/#CBP7501EntrySummaryCertificate">7501 Entry Summary</a> (1), presenting it to Customs (2), and associating it with the earlier submission by correlating the workflow instance id ("a123-b456").
```

File: docs/index.html [hunk 17]
```diff
@@ -955,21 +955,21 @@ <h5>Import Completion</h5>
...
+          Customs verifies the <a href="https://w3id.org/traceability/#CBP7501EntrySummaryCertificate">Entry Summary</a> (3) and completes the importing assessment. 
         </p>
         <h5>Trusted Trader Program</h5>
         <figure>
           <img src="resources/ctpat-workflow.png" alt="Trusted Trader Program Workflow" style="width:100%">
           <figcaption>Trusted Trader Program Workflow</figcaption>
         </figure>
         <p>
-          CTPAT runs vetting procedures, issues and presents a CTPAT Certificate (1) which is presented to the Importer (2).
+          CTPAT runs vetting procedures, issues and presents a <a href="https://w3id.org/traceability/#CTPATCertificate">CTPAT Certificate</a> (1) which is presented to the Importer (2).
```
Comment (TallTed) [inline docs/index.html:969]:
```suggestion
          CTPAT runs vetting procedures, and issues and presents to the Importer (2) a <a href="https://w3id.org/traceability/#CTPATCertificate">CTPAT Certificate</a> (1).
```

File: docs/index.html [hunk 18]
```diff
@@ -980,19 +980,19 @@ <h5>Freight Forwarder Consignment</h5>
           <figcaption>Freight Forwarder Consignment Workflow</figcaption>
         </figure>
         <p>
-          The product importer of record issues Proforma Invoice (1) and Intent To Import (2) credentials which is presented to CBP (3) with instance id [inst_1] well ahead of shipping. 
+          The product importer of record issues <a href="https://w3id.org/traceability/#ProformaInvoiceCertificate">Proforma Invoice</a> (1) and <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> (2) credentials which are presented to CBP (3) with instance id [inst_1] well ahead of shipping.
```
Comment (TallTed) [inline docs/index.html:983]:
```suggestion
          The product importer of record issues <a href="https://w3id.org/traceability/#ProformaInvoiceCertificate">Proforma Invoice</a> (1) and <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> (2), which are presented to CBP (3) with instance id ("inst_1") well ahead of shipping. 
```

File: docs/index.html [hunk 19]
```diff
@@ -980,19 +980,19 @@ <h5>Freight Forwarder Consignment</h5>
...
         <p>
-          The product importer of record issues Proforma Invoice (1) and Intent To Import (2) credentials which is presented to CBP (3) with instance id [inst_1] well ahead of shipping. 
+          The product importer of record issues <a href="https://w3id.org/traceability/#ProformaInvoiceCertificate">Proforma Invoice</a> (1) and <a href="https://w3id.org/traceability/#IntentToImportCertificate">Intent To Import</a> (2) credentials which are presented to CBP (3) with instance id [inst_1] well ahead of shipping. 
         </p>
         <p>
           CBP verifies the initial import documentation submitted by the Importer (4).
         </p>
         <p>
-          Upon shipping, the Freight Forwarder issues a House BL (5) which is presented to CBP (6) with a separate instance id [inst_2].
+          Upon shipping, the Freight Forwarder issues a <a href="https://w3id.org/traceability/#HouseBillOfLadingCertificate">House BL</a> (5) which is presented to CBP (6) with a separate instance id [inst_2].
```
Comment (TallTed) [inline docs/index.html:989]:
```suggestion
          Upon shipping, the Freight Forwarder issues a <a href="https://w3id.org/traceability/#HouseBillOfLadingCertificate">House BL</a> (5) which is presented to CBP (6) with a separate instance id ("inst_2").
```

File: docs/index.html [hunk 20]
```diff
@@ -980,19 +980,19 @@ <h5>Freight Forwarder Consignment</h5>
...
         </p>
         <p>
-          The Importer issues the final Commercial Invoice (8) which is presented to the Freight Forwarder (9) also with instance id [inst_1]. Passing on the Commercial Invoice, the Freight Forwarder includes both its own and the Importer’s instance ids [inst_1, inst_2], indicating to Customs that the workflows are related.
+          The Importer issues the final <a href="https://w3id.org/traceability/#CommercialInvoiceCertificate">Commercial Invoice</a> (8) which is presented to the Freight Forwarder (9) also with instance id [inst_1]. Passing on the <a href="">Commercial Invoice</a>, the Freight Forwarder includes both its own and the Importer’s instance ids [inst_1, inst_2], indicating to Customs that the workflows are related.
```
Comment (TallTed) [inline docs/index.html:992]:
```suggestion
          The Importer issues the final <a href="https://w3id.org/traceability/#CommercialInvoiceCertificate">Commercial Invoice</a> (8) which is presented to the Freight Forwarder (9) with instance id ("inst_1"). Passing on the <a href="">Commercial Invoice</a>, the Freight Forwarder includes both its own and the Importer’s instance ids ["inst_1", "inst_2"], indicating to Customs that the workflows are related.
```

**Comment by nissimsan** [comment]
Thanks a ton for these improvements, @TallTed!

**Review by BenjaminMoe — APPROVED** [review]

**Pull request merged**